payment-manager/src/test/java/com/qpay/paymentmanager/service/impl/PaymentServiceImplTest.java

```diff
@@ -81,6 +87,7 @@ void should_makePaymentBetweenWallets() {
         given(walletService.getWalletById(toId)).willReturn(toWallet);
         given(walletRepository.save(expectedFromWallet)).willReturn(expectedFromWallet);
         given(walletRepository.save(expectedToWallet)).willReturn(expectedToWallet);
+        doNothing().when(executorService).execute(any(Runnable.class));
```

Comment:
No, you should leave it. But instead of `doNothing` you should use bdd approach `willDoNothing`. Not `when` but `given` and so on. https://www.javaguides.net/2023/10/mockito-bddmockito-willdonothing.html